sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -158,6 +158,14 @@ object ScalaNativePluginInternal {
       }
       .tag(NativeTags.Link)
       .value,
+    console := console
```

Comment:
It might be nice if this was pushed to the bottom of these settings. Did it work for you? I tried it in my branch but had some problems but it might be due to other changes. When I type `console` it compiles everything and then fails.

Reply:
Weird, when I try this it just goes to a normal console. I put the code and the bottom of that block but I wouldn't think that would matter.

Reply:
The warning worked for me. Make sure to run this command in the scope of project with enabled ScalaNativePlugin, eg. `sandbox3/console`. When using `console` you would run it in the root project which does not enable this plugin